installer/scripts/tomlparser.rb

```diff
@@ -70,7 +71,7 @@ def populateSettingValuesFromConfigMap(parsedConfig)
         end
       end
     rescue => errorStr
-      puts "config::error::Exception while reading config settings for stdout log collection - #{errorStr}, using defaults"
+      ConfigParseErrorLogger.logError("Exception while reading config settings for stdout log collection - #{errorStr}, using defaults")
```

Comment:
please look into all messages and make it descriptive so its actionable for customers, if not route to our AI telemetry